sentry/src/main/java/io/sentry/SentryTransaction.java

```diff
@@ -104,8 +104,8 @@ public void setName(final @NotNull String name) {
    * @param parentSpanId - parent span id
    * @return a new transaction span
    */
-  @Override
-  public @NotNull Span startChild(final @NotNull SpanId parentSpanId) {
+  @NotNull
+  Span startChild(final @NotNull SpanId parentSpanId) {
```

Comment:
should all of them return `ISpan` instead?

Reply:
Not this one but it can be private.